@oleg-andreyev Thanks for pushing on this front. Looks like there is only 1 failure left?

```
There was 1 failure:

1) Elastica\Test\Aggregation\RangeTest::testRangeAggregation
Failed asserting that 3 matches expected 2.

/home/runner/work/Elastica/Elastica/tests/Aggregation/RangeTest.php:28

--

There were 6 skipped tests:

1) Elastica\Test\Cluster\SettingsTest::testSetTransient
discovery.zen.minimum_master_nodes is deprecated, ignored in 7.x and removed in 8.x, see: https://www.elastic.co/guide/en/elasticsearch/reference/master/migrating-8.0.html#breaking-changes-8.0

/home/runner/work/Elastica/Elastica/tests/Cluster/SettingsTest.php:21

2) Elastica\Test\Cluster\SettingsTest::testSetPersistent
discovery.zen.minimum_master_nodes is deprecated, ignored in 7.x and removed in 8.x, see: https://www.elastic.co/guide/en/elasticsearch/reference/master/migrating-8.0.html#breaking-changes-8.0

/home/runner/work/Elastica/Elastica/tests/Cluster/SettingsTest.php:47

3) Elastica\Test\IndexTest::testForcemerge
Failed asserting that 2 is identical to 0.

/home/runner/work/Elastica/Elastica/tests/IndexTest.php:863

4) Elastica\Test\Multi\SearchTest::testSearchWithError
Elastica\Exception\ResponseException: [size] parameter cannot be negative, found [-2]

/home/runner/work/Elastica/Elastica/tests/Multi/SearchTest.php:295

5) Elastica\Test\Multi\SearchTest::testSearchWithErrorWithKeys
Elastica\Exception\ResponseException: [size] parameter cannot be negative, found [-2]

/home/runner/work/Elastica/Elastica/tests/Multi/SearchTest.php:[34](https://github.com/ruflin/Elastica/actions/runs/4462337031/jobs/7836801245?pr=2123#step:7:35)0

6) Elastica\Test\SnapshotTest::testSnapshotAndRestore
Failed asserting that actual size 2 matches expected size 1.

/home/runner/work/Elastica/Elastica/tests/SnapshotTest.php:76

FAILURES!
Tests: [48](https://github.com/ruflin/Elastica/actions/runs/4462337031/jobs/7836801245?pr=2123#step:7:49)6, Assertions: 2094, Failures: 1, Skipped: 6.

Generating code coverage report in Clover XML format ... done [00:00.1[68](https://github.com/ruflin/Elastica/actions/runs/4462337031/jobs/7836801245?pr=2123#step:7:69)]
Error: Process completed with exit code 1.
```

--- a/docker/docker-compose.es.yml
+++ b/docker/docker-compose.es.yml
@@ -6,8 +6,6 @@ services:
         command: &command >
             /bin/sh -c "
               (./bin/elasticsearch-plugin list | grep -q ingest-attachment || ./bin/elasticsearch-plugin install --batch ingest-attachment) \
-              && mkdir -p /usr/share/elasticsearch/repository \
-              && chown -R elasticsearch: /usr/share/elasticsearch/repository \
               && /usr/local/bin/docker-entrypoint.sh
             "
         environment: &environment

--- a/tests/Aggregation/RangeTest.php
+++ b/tests/Aggregation/RangeTest.php
@@ -25,7 +25,7 @@ class RangeTest extends BaseAggregationTest
         $query->addAggregation($agg);
         $results = $this->_getIndexForTest()->search($query)->getAggregation('range');
 
-        $this->assertEquals(2, $results['buckets'][0]['doc_count']);
+        $this->assertEquals(3, $results['buckets'][0]['doc_count']);
     }
 
     /**

--- a/tests/Transport/HttpTest.php
+++ b/tests/Transport/HttpTest.php
@@ -168,7 +168,7 @@ class HttpTest extends BaseTest
 
         $this->assertEquals(0, $resultSet->getTotalHits());
 
-        $response = $index->request('/_search', 'POST');
+        $response = $index->request('_search', 'POST');
 
         $builder = new DefaultBuilder();
         $resultSet = $builder->buildResultSet($response, Query::create([]));

